src/test/java/com/example/bank/userRegister/UserServiceTest.java

```diff
@@ -0,0 +1,106 @@
+package com.example.bank.userRegister;
+
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.mockito.Mockito.*;
+
+//@SpringBootTest - komentarz dla mnie - nie używać tej adnotacji; ładuje cały kontekst aplikacji, czyli długo trwa!!
+@ExtendWith(MockitoExtension.class)
+class UserServiceTest {
+    private AutoCloseable closeable;
+    private static final User USER = User.builder()
+            .firstName("Zdzislaw")
+            .lastName("Krecina")
+            .email("zdzislaw.krecina@gmail.com")
+            .birthDate(LocalDate.of(1954, 4, 28))
+            .address(new Address("Tatrzanska", "12-456", "Zywiec"))
+            .build();
+    private static final UserRequest USER_REQUEST = UserRequest.builder()
+            .firstName("Zdzislaw")
+            .lastName("Krecina")
+            .email("zdzislaw.krecina@gmail.com")
+            .birthDate(LocalDate.of(1954, 4, 28))
+            .address(new Address("Tatrzanska", "12-456", "Zywiec"))
+            .build();
```

Comment:
takie rzeczy to do metod pomocnicznych wydziel

Reply:
już dawno zrobione